pkg/runtime-tools/generate/generate.go

```diff
@@ -103,6 +112,9 @@ func (g *Generator) Adjust(adjust *nri.ContainerAdjustment) error {
 	g.AdjustEnv(adjust.GetEnv())
 	g.AdjustHooks(adjust.GetHooks())
 	g.AdjustDevices(adjust.GetLinux().GetDevices())
+	if err := g.InjectCDIDevices(adjust.GetCDIDevices()); err != nil {
```

Comment:
Let's move this so that we're dealing with the `GetLinux()` adjustments separately / consecutively.

Reply:
Moved it accordingly.